lms/djangoapps/courseware/tabs.py

```diff
@@ -397,3 +408,18 @@ def _get_dynamic_tabs(course, user):
                 dynamic_tabs.append(tab)
     dynamic_tabs.sort(key=lambda dynamic_tab: dynamic_tab.name)
     return dynamic_tabs
+
+
+def _get_tabs_from_overview(course, user):
+    """
+    Returns the tabs updated in the Couse Overview.
+    """
+    overview_tabs = []
+    try:
+        course_overview = CourseOverview.objects.get(id=course.id)
```

Comment:
Should we use the `get_from_id` method instead, to take advantage of cache ?

```suggestion
        course_overview = CourseOverview.get_from_id(course.id)
```